frontend/src/container/NewWidget/utils.ts

```diff
@@ -419,7 +445,6 @@ export function handleQueryChange(
 					set(tempQuery, 'offset', 0);
 					set(tempQuery, 'pageSize', 10);
 				} else if (tempQuery.aggregateOperator === 'noop') {
-					set(tempQuery, 'aggregateOperator', 'count');
```

Comment:
Why was this added and does it break anything if we remove it now?

Reply:
when shifting from any panel other than list type we modify the operator to `noop` for list panel to work, we do not update the superset query here. so when moving back to any other panel from list panel the older operator should be selected by default we should not update it back to `count` here as it could have been anything.